eng/common/scripts/Verify-Links.ps1

```diff
@@ -115,15 +117,19 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 function CheckLink ([System.Uri]$linkUri)
 {
   if ($checkedLinks.ContainsKey($linkUri)) { return }
+  $badLinkEnabled = $false;
+
+  Write-Verbose "Checking link $linkUri..."  
 
-  Write-Verbose "Checking link $linkUri..."
   if ($linkUri.IsFile) {
     if (!(Test-Path $linkUri.LocalPath)) {
       LogWarning "Link to file does not exist $($linkUri.LocalPath)"
       $script:badLinks += $linkUri
+      $badLinkEnabled = $true
     }
   }
   else {
+
```

Comment:
nit: Lets remove this empty line so that the try block is at the top of the block.